.github/workflows/upstream.yml

```diff
@@ -0,0 +1,46 @@
+name: Upstream
+
+on: [push, pull_request]
+
```

Comment:
Maybe we could also let this run regularly by a corn job once a week to catch failings even if we don’t actively work on PRs?

Reply:
See xarray or climpred

Reply:
good idea. i'll base of of the climpred one and https://github.com/pydata/xarray/blob/master/.github/workflows/upstream-dev-ci.yaml